generated/nidaqmx/constants.py

```diff
@@ -881,10 +901,14 @@ class TriggerType(Enum):
     DIGITAL_EDGE = 10150  #: Trigger on a rising or falling edge of a digital pulse.
     DIGITAL_LEVEL = 10152  #: Pause the measurement or generation while a digital signal is at either a high or low state.
     DIGITAL_PATTERN = 10398  #: Pause the measurement or generation while digital physical channels either match or do not match a digital pattern.
+    EXTERNAL_DIGITAL_CLOCK = 10400
+    RATE = 10402
     ANALOG_EDGE = 10099  #: Trigger when an analog signal crosses a threshold.
     ANALOG_MULTI_EDGE = 16108  #: Trigger when any of the configured analog signals cross their respective thresholds.
+    VIDEO = 10476
```

Comment:
This looks like old internal constant... can we remove?

Reply:
Removed.